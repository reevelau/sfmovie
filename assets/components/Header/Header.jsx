import React, { Component } from 'react';

export default class Header extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div class="header">
        <h3>San Francisco Movies Locations</h3>
        <span class="description">Enter a movie name and we will show you the relating locations in the area.</span>
      </div>
    );
  }
}
