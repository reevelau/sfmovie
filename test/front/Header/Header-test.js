import React from 'react';
import Header from '../../../assets/components/Header/Header';
import renderer from 'react-test-renderer';


test('Normal', () => {

    const component = renderer.create(
        <Header />
    );
    
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});