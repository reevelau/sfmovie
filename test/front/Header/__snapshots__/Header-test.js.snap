// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Normal 1`] = `
<div
  className="header"
>
  <h3>
    San Francisco Movies Locations
  </h3>
  <span
    className="description"
  >
    Enter a movie name and we will show you the relating locations in the area.
  </span>
</div>
`;
